examples/particle/main.rs

```diff
@@ -4,51 +4,110 @@ use blade_graphics as gpu;
 
 mod particle;
 
+// const SAMPLE_COUNT: u32 = 1;
+// const SAMPLE_COUNT: u32 = 2;
+const SAMPLE_COUNT: u32 = 4;
+// const SAMPLE_COUNT: u32 = 8;
+
 struct Example {
     command_encoder: gpu::CommandEncoder,
     prev_sync_point: Option<gpu::SyncPoint>,
     context: gpu::Context,
     surface: gpu::Surface,
     gui_painter: blade_egui::GuiPainter,
     particle_system: particle::System,
+
+    msaa_texture: gpu::Texture,
+    msaa_view: gpu::TextureView,
 }
 
 impl Example {
+    fn resize(&mut self, size: winit::dpi::PhysicalSize<u32>) {
+        let config = Self::make_surface_config(size);
+        self.context.reconfigure_surface(&mut self.surface, config);
+
+        let surface_info = self.surface.info();
+
+        if let Some(sp) = self.prev_sync_point.take() {
+            self.context.wait_for(&sp, !0);
+        }
+        self.context.destroy_texture_view(self.msaa_view);
+        self.context.destroy_texture(self.msaa_texture);
+
+        // self.gui_painter.destroy(&self.context);
```

Comment:
is this commented out because in theory we'd want to recreate the pipelines?
I think, for the purpose of this example, we can just assert that the format hasn't changed. No need to keep the code commented out